instance/models/openedx_instance.py

```diff
@@ -154,6 +154,9 @@ def save(self, **kwargs):
         if self.random_prefix is not None:
             self.mysql_user = self.random_prefix
 
+        if self.database_name is None or self.database_name == '':
```

Comment:
nit: you could shorten this to `if not self.database_name:`

Reply:
Got it